packages/connect-node/src/protocol-handler.ts

```diff
@@ -85,8 +94,94 @@ export interface UniversalHandler extends UniversalHandlerFn {
   allowedMethods: string[];
 
   /**
-   * A regular expression that matches all Content-Type header values that this
-   * procedure supports.
+   * A matcher for Content-Type header values that this procedure supports.
    */
-  supportedContentType: RegExp;
+  supportedContentType: ContentTypeMatcher;
```

Comment:
`ContentTypeMatcher` allows us to use the same type for this property in individual handlers for a protocol, and in merged handlers that handle multiple protocols.